README.md

```diff
@@ -19,147 +17,166 @@ artifactId: `kastro`
 ```kotlin
 implementation("org.jetbrains.kotlinx:kotlinx-datetime:0.5.0")
 implementation("dev.jamesyox:kastro:0.1.0")
 ```
-to your dependencies and be able to use the library.
 
-## Cookbook
-Kastro leverages Kotlin Sequences to lazily evaluate the algorithms contained within. This means that you can determine
-what time the sun will rise tomorrow, or every sunrise time for the next 10 years with a single call. I think a few
-examples will make it clear how powerful this approach is. You can use all the power of Kotlin's Sequences (stdlib) to
-answer all sorts of questions. See `ReadmeExamples.kt` for a compiled full text version of all of these.
+## Solar Events
+
+You can calculate the following solar events:
+
+| Event Name | Phase | Description |
+|-----:|---------------|-----------|
+|     Astronomical Twilight| Dawn, Dusk | For `AstronomicalDawn`, sun is increasing in brightness and is transitioning from `Night` toward `NauticalDawn`. For `AstronimcalDusk`, Sun is decreasing in brightness and is transitioning from `NauticalDusk` toward `Night`|
+|     Blue Hour| Dawn, Dusk | For `BlueHourDawn`, Sun is increasing in brightness and is transitioning between `NauticalDawn` toward `CivilDawn`. For `BlueHourDusk`, Sun is decreasing in brightness and is transitioning between `CivilDusk` toward `NauticalDusk`|
+|     Civil Twilight| Dawn, Dusk| For `CivilDawn`, sun is increasing in brightness and is transitioning from `NauticalDawn` toward `Day`. For `CivilDusk`, Sun is decreasing in brightness and transitioning from `Day` toward `NauticalDusk`|
+|     Day| Dawn| Sun is above the horizon|
+|     Golden Hour| Dawn, Dusk| For `GoldenHourDawn`, Sun is increasing in brightness and is transitioning between `CivilDawn` toward `Day`. For `GoldeHourDusk`, Sun is decreasing in brightness and is transitioning between `Day` toward `CivilDusk`|
+|     Nadir| Dusk| Opposite of Noon. Sun is at its lowest point below the horizon|
+|     Nautical Twilight| Dawn, Dusk| For `NauticalDawn`, sun is increasing in brightness and is transitioning from `Night` toward `CivilDawn`. For `NauticalDusk`, Sun is decreasing in brightness and is transitioning from `CivilDusk` toward `Astronomical`|
+|     Night | Dusk | Sun is below the horizon|
+|     Noon| Dawn| Opposite of Nadir. Sun is at its highest point above the horizon|
+|     Sunrise| Dawn| Sun's top edge first rises above the horizon|
+|     Sunset| Dusk| Sun's top edge completely disappears below the horizon|
+
+## Lunar Events
+
+| Event Name | Description |
+|-----:|---------------|
+|     First Quarter | Moon is increasing in brightness and is transitioning toward `FullMoon`|
+|     Full Moon | Moon is fully illuminated|
+|     Last Quarter | Moon is decreasing in brightness and is transitioning toward `NewMoon`|
+|     New Moon | Moon is not illuminated|
+|     Waning Crescent | Moon is decreasing in brightness and and is transitioning between `FullMoon` and `LastQuarter` |
```

Comment:
I think Waning and Waxing are reversed here :smile: Waning Crescent is a moon state between `LastQuarter` and `NewMoon` and `Waning Gibbous` is between `FullMoon` and `LastQuarter`.

Also I think that there are not exactly events but states of the moon. The only real "Phase Events" in Kastro are `NewMoon` `FirstQuarter` `FullMoon` `LastQuarter`

States like `WaningGibbous` occur between those events. It's safe to say the moon is always going to be in a state of `WaningGIbbous` `WaningCrescent` `WaxingGibbous` or `WaxingCrescent` even if we are really close to the true `Instant` in time that an event represents. A `FullMoon` isn't so much a state but a moment in time that we can roughly calculate. I didn't want to assume what range someone might consider the moon to be in the state of `Full Moon` as there's no established standard for that.

We might want two tables. Or one table that has all the states and the events that begin and end those states (like for the Solar Events).

Reply:
Swapped Crescent & Gibbous 🤦‍♀️ 

NASA refers to the waxing/waning gibbous/crescent states as "phases" in the same way they refer to first/last quarter and new/full moon, as far as I can tell. I think for now that stating everything the library can do in an easily digestible format is fine, and may make things less confusing.  I'm happy to be proven wrong later down the line, but I think for now this can let the user know what is supported in this library and an overview of what it means. 

Maybe for now this could be resolved by renaming the table `Lunar Phases` instead of `Lunar Events`?